demo-extended/build.gradle

```diff
@@ -9,13 +9,13 @@ buildscript {
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
 
     defaultConfig {
         applicationId "com.novoda.downloadmanager.demo.extended"
         minSdkVersion 14
-        targetSdkVersion 25
+        targetSdkVersion 26
```

Comment:
The apps work fine targeting either 25 or 26, I left them as 26 to make the build.gradle files free of warnings.